nextcord/client.py

```diff
@@ -632,6 +636,13 @@ def run(self, *args: Any, **kwargs: Any) -> None:
             is blocking. That means that registration of events or anything being
             called after this function call will not execute until it returns.
         """
+        
+        for libraryName in ["discord", "discord_slash", "pyfork", "enhanced-dpy"]: #more imcompatable libraries to be added
```

Comment:
This should be snake case

Reply:
pyfork and discord.py should be added

Reply:
The comment should be removed

Reply:
> why did you close this btw?

renamed branch due to spelling error in branch name